setup.cfg

```diff
@@ -22,3 +22,10 @@ exclude=pydev,resources.py,conf.py,third_party,ui,help,external,utils/importer,t
 #   http://pycodestyle.pycqa.org/en/latest/intro.html#error-codes
 ignore=E121,E123,E126,E133,E203,E226,E241,E242,E704,E501,W503,W504,E402,W605
 max-line-length = 88
+
+
+[flake8]
```

Comment:
Note: I moved `--cov` from pytest's default arguments to `make test`, likewise I wanted to add `--flake8` to the `make test` target.

But... I have to fix quite a number of unused imports and so before I can switch it on. That'll be a different PR :-)